vars/unitPackages.groovy

```diff
@@ -0,0 +1,41 @@
+// vars/unitPackages.groovy
+
+/**
+ * unitPackages.groovy
+ *
+ * unitPackages variable
+ */
+
+/**
+ * Method to return the list of Unit Testing packages
+ */
+
+String call() {
+    Map stage_info = parseStageInfo()
+    if (stage_info['target'] == 'centos7') {
+        String packages =  'gotestsum openmpi3 ' +
+                           'hwloc-devel argobots ' +
+                           'fuse3-libs fuse3 ' +
+                           'boost-devel ' +
+                           'libisa-l-devel libpmem ' +
+                           'libpmemobj protobuf-c ' +
+                           'spdk-devel libfabric-devel '+
+                           'pmix numactl-devel ' +
+                           'libipmctl-devel python36-pyxattr ' +
+                           'python36-tabulate numactl ' +
+                           'libyaml-devel ' +
+                           'valgrind-devel patchelf'
```

Comment:
Again, should this be provided by a file in the DAOS repo?

Reply:
Fixed in next push.